oak_attestation_verification/src/verifier.rs

```diff
@@ -517,14 +517,27 @@ fn verify_kernel_layer(
     )
     .context("kernel failed verification")?;
 
-    verify_regex(
-        values.kernel_raw_cmd_line.as_str(),
-        reference_values
-            .kernel_cmd_line_regex
-            .as_ref()
-            .context("no kernel command line regex reference values")?,
-    )
-    .context("kernel command line failed verification")?;
+    if let Some(kernel_raw_cmd_line) = values.kernel_raw_cmd_line.as_ref() {
+        verify_regex(
+            kernel_raw_cmd_line.as_str(),
+            reference_values
+                .kernel_cmd_line_regex
+                .as_ref()
+                .context("no kernel command line regex reference values")?,
+        )
+        .context("kernel command line failed verification")?;
+    } else {
+        // Support missing kernel_cmd_line_regex but only if the corresponding reference
```

Comment:
FWIW this still feels backwards to me (i.e. we should start from the expected reference value). I know we agreed to clean it up later on anyway :)

Reply:
Ack :)